api/routes_rtfsp.go

```diff
@@ -600,6 +597,49 @@ func (api *API) createHostedIPFSNetworkEntryInDatabase(c *gin.Context) {
 	})
 }
 
+func (api *API) startIPFSPrivateNetwork(c *gin.Context) {
+	username := GetAuthenticatedUserFromContext(c)
+	networkName, exists := c.GetPostForm("network_name")
+	if !exists {
+		FailWithMissingField(c, "network_name")
+		return
+	}
+	logger := api.LogWithUser(username).WithField("network_name", networkName)
+	logger.Info("private ipfs network start requested")
+
+	// verify access to the requested network
+	networks, err := api.um.GetPrivateIPFSNetworksForUser(username)
```

Comment:
we might want a helper function to do this check, I can put in a PR later one this lands

Reply:
Sounds good